pymc3/distributions/multivariate.py

```diff
@@ -383,10 +383,9 @@ class Dirichlet(Continuous):
 
     .. math::
 
-       f(\mathbf{x}) =
-           \frac{\Gamma(\sum_{i=1}^k \theta_i)}{\prod \Gamma(\theta_i)}
-           \prod_{i=1}^{k-1} x_i^{\theta_i - 1}
-           \left(1-\sum_{i=1}^{k-1}x_i\right)^\theta_k
+       f(\mathbf{x}|\mathbf{a}) =
+           \frac{\Gamma(\sum_{i=1}^k a_i)}{\prodi_{i=1}^k \Gamma(a_i)}
```

Comment:
`\prodi_{i=1}`: you're missing a space there (right after \prod)

Reply:
You are right! Thank you for checking that! Fixed it in the commit below.